HW15/src/FileLogger.java

```diff
@@ -0,0 +1,32 @@
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileLogger {
+
+    private final FileLoggerConfiguration fileLoggerConfiguration;
+
+
+    public FileLogger(){
+        fileLoggerConfiguration = new FileLoggerConfiguration();
+    }
+
+
+    public void info(String message) throws IOException, FileMaxSizeReachedException {
```

Comment:
IOException should not be here - probably, this exception must have been catched and processed earlier.